nibabel/parrec.py

```diff
@@ -206,6 +206,14 @@
     'Max. number of gradient orients': ('max_gradient_orient', int),
     # Line below added for par / rec version > 4.1
     'Number of label types   <0=no ASL>': ('nr_label_types', int),
+
```

Comment:
Vague preferernce for removing blank line here, comments do enough of a job of spacing.